### PR TITLE
khong hien cac job chua dc admin chap thuan

### PR DESCRIPTION
**Pull request by nmt0607**
## Related Tickets:
- [#33047](https://edu-redmine.sun-asterisk.vn/issues/33047)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by nmt0607** [comment]
ready

File: app/Http/Controllers/JobController.php [hunk 1]
```diff
@@ -16,14 +16,14 @@ public function index()
     {
         $tag = Auth::user()->tags->where('type', config('tag_config.skill'))->first();
         if (is_null($tag)) {
-            $suitableJobs = Job::with('images')->orderBy('created_at', 'desc')->with('tags')->get();
+            $suitableJobs = Job::where('status', config('job_config.approve'))->with('images')->orderBy('created_at', 'desc')->with('tags')->get();
```
Comment (congnx-2423) [inline app/Http/Controllers/JobController.php]:
```suggestion
        $suitableJobs = $tag->jobs()->where('status', config('job_config.approve'))->get();
        if (is_null($tag) {
            $suitableJobs = Job::where('status', config('job_config.approve'))->with('images')->orderBy('created_at', 'desc')->with('tags')->get();
        }
```

**Comment by nmt0607** [comment]
ready

**Comment by nmt0607** [comment]
ready

**Pull request merged**